docs/home/docs-contribution-guide.md

```diff
@@ -0,0 +1,88 @@
+# Installation Guide: Running the Server Locally
+
+#### Prerequisites:
+
+- Node.js installed on your machine. You can download it from
+  [the official Node.js website](https://nodejs.org/).
+- Yarn package manager installed. You can install it by following the instructions below:
+
+#### Steps (Assuming you have Node.js Installed):
+
+1. **Install Yarn:**
+
+   - Download and install Yarn by following the instructions on the
+     [Yarn website](https://classic.yarnpkg.com/lang/en/docs/install/).
+     - You can also install Yarn globally using npm:
+     ```
+     npm install -g yarn
+     ```
+   - After installing, you can verify the installation by running:
+     ```
+     yarn --version
+     ```
+   - This should display the installed Yarn version.
+
+2. **Fork the Project Repository:**
+
+   - Fork the project repository to your GitHub account by clicking the "Fork" button at the top
+     right of the repository page.
+
+3. **Clone the Project Repository:**
+
+   - Clone the project repository to your local machine using Git:
+     ```
+     git clone https://github.com/your-username/project-repository-name.git
+     ```
```

Comment:
this seems like a generic guide? Needs to clone docs.

Reply:
will effect the changes shortly, thanks for the feedback

Reply:
@jthiller would preciate your review feedback 😊